### DISCUSSION TITLE
How to keep old static bundles on every deployment?

### DISCUSSION THREAD

**Discussion by jnm733** [vercel/next.js#61712] [Help]
### Summary

Hi all!

I'm deploying my Next.js app with Kubernetes and I'm containerizing my app in a Docker image.

The problem is that on every build, the static folder content is replaced by the new bundles with different names. Then when I enter on a browser cached url, it tries to load old bundles that do not exist now and the page loads without styles and interactivity.

Which are the available options to keep older bundles on every build or use the same syntax used in dev mode that simply adds a timestamp on the get parameter to the same filename

_next/static/css/app/layout.css?v=1707214117133 

Thanks!

**Comment by Oleg55** [comment]
up please, same issue
